### DISCUSSION TITLE
Export markdown without linebreaks in paragraphs

### DISCUSSION THREAD

**Discussion by joel-marks** [Zettlr/Zettlr#5117] [Q&A] [answered]
Hi there, I'm trying to solve an issue I encounter with exporting markdown files.

On export my paragraph text has line breaks inserted as if in a word processor with a page-width. Only  unbreakable strings such as URLs are exempt.

I wish to use the output in a website or other editor which will deal with word-wrapping so the export inserting carriage returns is a real issue.

I'm exporting "Markdown gfm+raw_html" as it is the only export recipe that says "Markdown."

Using Kubuntu 23.10, Plasma 5.27.8, Zettlr Zettlr-3.0.5-x86_64.AppImage, UUID: a3de68d3-90c2-41e8-8d77-aa5532c6034c and Zotero for references which is very effective and working well.

Any support appreciated. I realise this is probably a simple fix in the export recipe file but I'm having no luck searching the answer.

**Comment by nathanlesage** [comment]
> I wish to use the output in a website or other editor which will deal with word-wrapping so the export inserting carriage returns is a real issue.

To view any file you create with Zettlr in another editor, right-click the file and open it with whatever editor you want. To copy a file's content into some website (e.g., as a blog post), simply do that: copy and paste.

"Exporting" a Markdown file to Markdown in Zettlr effectively only reformats the files without doing much else; and one of these reformatting rules is to change the column width to 80 characters at most.

**Reply by joel-marks** [reply]
Hi there. Understood. But the point of exporting for me is that the export pulls in the references from Zotero and populates the page with a reference list.

**Reply by nathanlesage** [reply, marked as answer]
> But the point of exporting for me is that the export pulls in the references from Zotero and populates the page with a reference list.

That is an important detail that you forgot to mention, but that is correctly one of the reformats that is happening. Please refer to the [Pandoc manual](https://pandoc.org/MANUAL.html#general-writer-options) on how to disable wrapping:

```yaml
wrap: preserve # or: none
```

@sensologica @wideaperture Would this be something that might make sense to include in the default defaults files?

**Reply by joel-marks** [reply]
Thank you so much. Yes. Problem solved. Just added the above

```wrap: none```

to the last line of the export recipe file I was using in

```.config/Zettlr/defaults/Markdown.yaml```

...I did for a moment get confused with the export section of the settings dialogue. It was easier to just edit the yaml file.
